csv_convert/rigol.py

```diff
@@ -0,0 +1,32 @@
+"""CSV file convert.
+
+This is a python excution file to convert RIGOL csv file
+into HummingBird readable format.
+
+"""
+import argparse
+import csv
+import os
+
+
+if __name__ == "__main__":
+  parser = argparse.ArgumentParser()
+  parser.add_argument("input_csv", help="input csv data path")
+  parser.add_argument("--output_csv", default=None, help="output csv data path")
+  args = parser.parse_args()
+
+  if args.output_csv is None:
+    args.output_csv = args.input_csv[:-4]+"_converted.csv"
+
+  if os.path.isfile(args.input_csv):
```

Comment:
I think this line is redundant, open(...) or csv.reader(...) will fail if the path is something else..